include/nbl/asset/IImage.h

```diff
@@ -232,9 +238,10 @@ class IImage : public IDescriptor
 			E_SAMPLE_COUNT_FLAGS						samples;
 			// stuff below is irrelevant in OpenGL backend
 			E_TILING									tiling = static_cast<E_TILING>(0);
```

Comment:
@Erfan-Ahmadi @achalpandeyy suggestions for a default value?

Reply:
that doesn't impact our ability to transfer in and out of them?

Reply:
I'm not sure didn't find anything in the spec, there is only 2 options for a default and OPTIMAL seems like a better choice to me.

Reply:
ok change it.